src/lib/bakeryjs/Box.ts

```diff
@@ -332,8 +332,18 @@ abstract class Box extends EventEmitter implements BoxInterface {
 			);
 
 			guardedQ.revoke();
-			this.queue.push(value.createSentinel(siblingsCount, retValue));
-			return;
+			this.emit('generation_finished', [
+				Object.create(null, {
```

Comment:
Why do you need Object.create? Why not just pass an empty object?

Reply:
Fixed.